cluster/aws/util.sh

```diff
@@ -721,14 +723,11 @@ function create-iam-profile {
 
 # Creates the IAM roles (if they do not already exist)
 function ensure-iam-profiles {
-  aws iam get-instance-profile --instance-profile-name ${IAM_PROFILE_MASTER} || {
```

Comment:
Isn't that dangerous? I am not entirely sure if we want to delete it without any warning or an explicit confirmation from the user. Note that, kube-up/kube-down are not just used in tests. It is used in real production setups by users.